docs/requirements.txt

```
@@ -5,5 +5,5 @@
 # the project.
 
 sphinx >=3.0, <5.0
-sphinx-rtd-theme >=0.5, <1.0
+sphinx-rtd-theme >=0.5, <2.0
 readthedocs-sphinx-search >=0.1, <1.0
```

Remove minimum requirement
```suggestion
readthedocs-sphinx-search <1.0
```